holoviews/tests/element/test_tiles.py

```diff
@@ -107,7 +107,7 @@ def test_pandas_series(self):
         self.check_array_type_preserved(
             pd.Series, pd.Series,
             lambda a, b: pd.testing.assert_series_equal(
-                a, b, check_exact=False, check_less_precise=True,
```

Comment:
Any idea what `check_less_precise` was about?

Reply:
Not precisely (haha sorry). It'd defined [here](https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html) and given its name I assumed that if the test passes without it, then it should be safe.